app/models/challenge.rb

```diff
@@ -32,11 +32,11 @@ def featured_contributions
     when 'experience'
       experience_stage.experiences.order_by('created_at DESC').published.first(2)
     when 'idea'
-      idea_stage.ideas.published.where(inspiration: false).first(3)
+      idea_stage.ideas.published.order_by('created_at DESC').where(inspiration: false).first(3)
```

Comment:
@stedmanblake can we do this on a diff PR? As a general comment, let's try to keep the contents of the PRs close to what their title is / down to one commit. E.g. I would have been happy to merge this, but have some thoughts on this part here. Namely, that I think this will get rid of the other scopes you've defined (e.g. published) if we go this route? I think we should think about it a tad bit more to make sure we're doing the right scoping / ordering here.